react-function-components/src/index.jsx

```diff
@@ -0,0 +1,6 @@
+import React from "react";
+import ReactDom from "react-dom";
+
+const CustomButton = <button>Click Me!</button>;
```

Comment:
This is not a function component good sirl. Please refactor with a function component